sim/midas/src/main/cc/simif_emul.cc

```diff
@@ -30,6 +30,14 @@ double sc_time_stamp() { return (double)main_time; }
 extern void tick();
 #endif // VCS
 
+mmio_t *simif_emul_t::master = new mmio_t(CTRL_BEAT_BYTES);
```

Comment:
So did you have to define these because verilator is using a C++ std older than c++17?  The next verilator version we're pulling in should fix that.

Reply:
I consider this to be a C++ anti-pattern: if you define static vars in a header, you get weak copies of it in all compilation units including the header and the linker has to de-duplicate. This leads to all sort of fun problems if you accidentally compile different compilation units with different flags.